objects/ocamera/Step_2.gml

```diff
@@ -1,13 +1,22 @@
 /// @description Insert description here
 // You can write your code in this editor
 
-if (p1 != noone && p2 != noone && p1.state != eState.HITSTOP && p2.state != eState.HITSTOP)
+// Set the camera's target
+if (p1 != noone && p2 != noone)

```

Comment:
This if statement should be nested under the one below ```if (room != rStageArcade)``` as the value being set is only ever used when that condition is hit.